iac-extensions/arm/src/test/java/org/sonar/iac/arm/checks/UnencryptedCloudServicesCheckTest.java

```diff
@@ -159,13 +214,35 @@ void testResourcesWithInfrastructureEncryption(String resourceType) {
       issue(52, 14, 52, 16 + resourceTypeLength, "Omitting \"infrastructureEncryption\" enables clear-text storage. Make sure it is safe here."));
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = {
+    "Microsoft.DBforMySQL/servers",
+    "Microsoft.DBforPostgreSQL/servers"
+  })
+  void testResourcesWithInfrastructureEncryptionBicep(String resourceType) {
+    String content = ArmTestUtils.readTemplateAndReplace("UnencryptedCloudServicesCheck/MultiUnencryptedInfrastructureEncrypted_template.bicep", resourceType);
+
+    int resourceTypeLength = resourceType.length();
+    BicepVerifier.verifyContent(content, check,
+      issue(18, 4, 18, 31, "Make sure that using unencrypted cloud storage is safe here."),
+      issue(25, 4, 25, 25, "Make sure that using unencrypted cloud storage is safe here."),
+      issue(29, 48, 29, 80, "Omitting \"encryptionState\" enables clear-text storage. Make sure it is safe here."),
+      issue(36, 4, 36, 40, "Make sure that using unencrypted cloud storage is safe here."),
+      issue(40, 52, 40, 52 + resourceTypeLength, "Omitting \"infrastructureEncryption\" enables clear-text storage. Make sure it is safe here."));
```

Comment:
I believe you can still specify the issue directly in the bicep file and not provide the list of issues here, would be easier to read afterward (and also probably to setup).

Reply:
Actually, the way we are using `SingleFileVerifier` in `verifyConent` now doesn't seem to allow this: we never call `SingleFileVerifier#assertOneOrMoreIssues` from inside `verifyContent`